core/expressionParser/helpers/functionImplementations.js

```diff
@@ -0,0 +1,19 @@
+import * as A from '@core/arena'
+import * as DateUtils from '@core/dateUtils'
+
+import { functionNames } from './functions'
+
+export const functionImplementations = {
```

Comment:
can we just add a property called ```impl``` to the object functions ?

Reply:
we can, but if you want to pass a custom implementation, like I do from recordExpressionParser, you will have to pass the entire function descriptor.